### DISCUSSION TITLE
Want to launch Browser with Throttling Speed configurable (Slow 3G) for some case and for rest default (Fast 3G) or no throttling.

### DISCUSSION THREAD

**Discussion by Prashant-Kan** [cypress-io/cypress#27513] [General]
We have a scenario in which we want to ensure that the functionality should work under Slow 3G network speed too. For that I was looking for the browser configuration during its launching when test case starts. 

Let me know if anyone has any idea or have done this kind of Automation testing already sometime.

**Comment by grandEL-dsouza** [comment]
Hey @Prashant-Kan 

As per the resources online Network throttling can be achieved natively by Chrome (CDP) by using Network.emulateNetworkConditions.
Here’s a blog that I found online to show an example for network throttling:
https://chromedevtools.github.io/devtools-protocol/1-3/Network/#method-emulateNetworkConditions
Furthermore you can use this link as a reference:
https://www.cypress.io/blog/2020/11/12/testing-application-in-offline-network-mode#going-online

While this should ideally help you, please be informed that you can export the key 'CYPRESS-NETWORK-THROTTLE' to true as a key in your environment variable and use this Javascript:
<img width="529" alt="image" src="https://github.com/cypress-io/cypress/assets/60457320/fe9cedc5-8ff5-419a-9929-e598f073865f">
This should help you achieve your use case. 

Thanks.

**Reply by cachho** [reply]
In cypress `13.6.6` I get: `Property 'chrome' does not exist on type 'AUTWindow'.ts(2339)` - any idea how your snippet looks in a later version?

**Reply by Prashant-Kan** [reply]
Facing the same Issue with the latest version of cypress
